src/index.html

```diff
@@ -12,6 +12,8 @@
 
 	<!-- inject:css -->
 	<!-- endinject -->
+
+	<script src="/js/app.config.js"></script>
```

Comment:
I think the problem here is that once the app is compiled, the output is called `/js/app.js` and not `/js/app.config.js`.